notifications/util.go

```diff
@@ -72,3 +74,7 @@ func timeToEpoch(time time.Time) string {
 func timeToDate(time time.Time) string {
 	return time.Format("2006-01-02")
 }
+
+func timeToTime(time time.Time) string {
+	return time.Format("15:04:05 MST")
```

Comment:
Just one note. I suggest either using UTC, making the timezone configurable, or using the machine's one.

Reply:
@worldtiki No this is just how the https://golang.org/pkg/time/#Time.Format works. It's a reference time. You give it the `formatting` of how you want things to appear and it will use the correct timezone when it actually prints the value :)

Reply:
Oh right, sorry.
I always get confused with this unusual format that the go folks decided to use vs the standard yyyy-MM-dd etc.